common/lib/capa/capa/capa_problem.py

```diff
@@ -978,7 +974,7 @@ def response_a11y_data(self, response, inputfields, responsetype_id, problem_dat
             for description in description_tags:
                 descriptions[
                     "description_%s_%i" % (responsetype_id, description_id)
-                ] = description.text
+                ] = HTML(stringify_children(description))
```

Comment:
You can do this.  This won't have any affect until the template itself is safe by default through the page, in which case this HTML call will protect this from being escaped.

Again, you could use bleach if you want to make sure that it is safe HTML (limited to your definition of safe tags).  Otherwise, you may want to comment that this is assuming anything author inputs is ok.
@nedbat @nasthagiri: This issue keeps coming up.  Is it ok to leave any course author field open to any HTML if we want to support any HTML?  Otherwise, should we define somewhere what is OK, or use default bleach config?